lucet-runtime/lucet-runtime-internals/src/context/mod.rs

```diff
@@ -5,12 +5,12 @@ mod tests;
 
 use crate::instance::Instance;
 use crate::val::{val_to_reg, val_to_stack, RegVal, UntypedRetVal, Val};
-use failure::Fail;
 use nix;
 use nix::sys::signal;
 use std::arch::x86_64::{__m128, _mm_setzero_ps};
 use std::ptr::NonNull;
 use std::{mem, ptr};
+use thiserror::Error;
 use xfailure::xbail;
```

Comment:
We can now replace uses of `xfailure::xbail` with `anyhow::bail`, since it does the implicit type conversion of errors that I believe was @jedisct1's original motivation for `xfailure`.

Reply:
Thanks @acfoltzer!  You and I talked through this in person and identified that the one moment that `bail!` is used in this file may not be an appropriate moment to bail.  If the function was supplied a misaligned stack, let's just return an error indicating as such.